### PR TITLE
chore: add color resource ✨

### LINKED ISSUE #1027: [Add]: COLOR RESOURCE 💡
### Site URL

https://farbvelo.elastiq.ch/

### Description of the above provided site

- FarbVélo is a playful color picking tool. 
- It can come up with the most pleasing color combinations.

### Category

Frontend

### Sub Category

colors

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by 0xabdulkhalid**
## Fixes Issue

Closes #1027 

## Changes proposed
- Added `FarbVélo` resource to Frontend -> colors category.

## Screenshots

Screenshot 2023-06-16 at 20-56-12 LinksHub A hub of ready-to-use tech resources

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
LGTM

File: database/frontend/colors.json [hunk 1]
```diff
@@ -131,5 +131,12 @@
     "url": "https://vanschneider.com/colors",
     "category": "frontend",
     "subcategory": "colors"
+  },
+  {
+    "name": "FarbVélo",
+    "description": "A playful color picking tool which can generate most pleasing color combinations.",
```
Comment (rupali-codes) [inline database/frontend/colors.json:137]:
```suggestion
    "description": "A playful color picking tool that can generate most pleasing color combinations.",
```

**Comment by 0xabdulkhalid** [comment]
Changes have been made @rupali-codes

**Review by rupali-codes — APPROVED** [review]
thanks, lgtm

**Pull request merged**